commit f59b6634bd82af4dcd26b661784b591c8a561bfd
Author: Mehmet <mmt@mmt.com>
Date:   Tue Dec 30 15:55:14 2025 +0300

    Apply z-mozbuild-simdjson.patch

diff --git a/browser/app/moz.build b/browser/app/moz.build
index ac02abec0fc0..1a139b9e740f 100644
--- a/browser/app/moz.build
+++ b/browser/app/moz.build
@@ -182,4 +182,7 @@ else:
     WINCONSOLE = False
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/docshell/base/moz.build b/docshell/base/moz.build
index 0f73f2dd8203..9e14e8143732 100644
--- a/docshell/base/moz.build
+++ b/docshell/base/moz.build
@@ -125,4 +125,7 @@ EXTRA_JS_MODULES += ["URIFixup.sys.mjs"]
 
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
 
-LOCAL_INCLUDES += ['/camoucfg']
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/base/moz.build b/dom/base/moz.build
index 0598d58bfba3..53c4b192a53a 100644
--- a/dom/base/moz.build
+++ b/dom/base/moz.build
@@ -652,4 +652,7 @@ GeneratedFile(
 )
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/battery/moz.build b/dom/battery/moz.build
index 91d673039bcb..4bcb19535746 100644
--- a/dom/battery/moz.build
+++ b/dom/battery/moz.build
@@ -23,4 +23,7 @@ MOCHITEST_CHROME_MANIFESTS += ["test/chrome.toml"]
 MOCHITEST_MANIFESTS += ["test/mochitest.toml"]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/canvas/moz.build b/dom/canvas/moz.build
index 93fb0f513cf9..e15998a74258 100644
--- a/dom/canvas/moz.build
+++ b/dom/canvas/moz.build
@@ -231,4 +231,7 @@ if CONFIG["CC_TYPE"] == "gcc":
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/geolocation/moz.build b/dom/geolocation/moz.build
index 55bae7754e7d..b37355d94fb0 100644
--- a/dom/geolocation/moz.build
+++ b/dom/geolocation/moz.build
@@ -64,6 +64,7 @@ LOCAL_INCLUDES += [
     "/camoucfg",
     "/dom/base",
     "/dom/ipc",
+    "/third_party/simdjson/include",
 ]
 
 if CONFIG["MOZ_WIDGET_TOOLKIT"] == "android":
diff --git a/dom/media/moz.build b/dom/media/moz.build
index 9fe37f8ed6d9..b56edef66d03 100644
--- a/dom/media/moz.build
+++ b/dom/media/moz.build
@@ -424,4 +424,7 @@ FINAL_LIBRARY = "xul"
 SPHINX_TREES["/media"] = "docs"
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/dom/media/webaudio/moz.build b/dom/media/webaudio/moz.build
index 6a03f45bf7e4..8aaa1a2f0dbc 100644
--- a/dom/media/webaudio/moz.build
+++ b/dom/media/webaudio/moz.build
@@ -156,4 +156,7 @@ FINAL_LIBRARY = "xul"
 LOCAL_INCLUDES += [".."]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/media/webrtc/jsapi/moz.build b/dom/media/webrtc/jsapi/moz.build
index fb9c1133316d..d4ec4504c11d 100644
--- a/dom/media/webrtc/jsapi/moz.build
+++ b/dom/media/webrtc/jsapi/moz.build
@@ -69,4 +69,7 @@ EXPORTS.mozilla.dom += [
 FINAL_LIBRARY = "xul"
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/media/webspeech/synth/moz.build b/dom/media/webspeech/synth/moz.build
index e238b8d41847..5667c8afc81b 100644
--- a/dom/media/webspeech/synth/moz.build
+++ b/dom/media/webspeech/synth/moz.build
@@ -64,4 +64,7 @@ LOCAL_INCLUDES += [
 ]
 
 # DOM Mask
-LOCAL_INCLUDES += ['/camoucfg']
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/workers/moz.build b/dom/workers/moz.build
index b8ad10403ff3..e5c4492dc628 100644
--- a/dom/workers/moz.build
+++ b/dom/workers/moz.build
@@ -116,4 +116,7 @@ XPCSHELL_TESTS_MANIFESTS += ["test/xpcshell/xpcshell.toml"]
 BROWSER_CHROME_MANIFESTS += ["test/browser.toml"]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/gfx/src/moz.build b/gfx/src/moz.build
index faa0c113bc51..02047602b86e 100644
--- a/gfx/src/moz.build
+++ b/gfx/src/moz.build
@@ -97,4 +97,7 @@ if CONFIG["MOZ_WIDGET_TOOLKIT"] == "gtk":
     CXXFLAGS += CONFIG["MOZ_PANGO_CFLAGS"]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/gfx/thebes/moz.build b/gfx/thebes/moz.build
index 7efb2bb8f7a1..a24ae47123b2 100644
--- a/gfx/thebes/moz.build
+++ b/gfx/thebes/moz.build
@@ -301,4 +301,7 @@ CXXFLAGS += ["-Werror=switch"]
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/intl/components/moz.build b/intl/components/moz.build
index 72213b13cf69..cb87edff4021 100644
--- a/intl/components/moz.build
+++ b/intl/components/moz.build
@@ -87,4 +87,7 @@ DEFINES["MOZ_HAS_MOZGLUE"] = True
 Library("intlcomponents")
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/layout/style/moz.build b/layout/style/moz.build
index 1b13537577e7..811dc67ae723 100644
--- a/layout/style/moz.build
+++ b/layout/style/moz.build
@@ -365,4 +365,7 @@ if CONFIG["COMPILE_ENVIRONMENT"]:
     )
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/mozglue/moz.build b/mozglue/moz.build
index 4ef610890b2f..61b63a14e04a 100644
--- a/mozglue/moz.build
+++ b/mozglue/moz.build
@@ -26,4 +26,7 @@ DIRS += [
 if CONFIG["MOZ_WIDGET_TOOLKIT"]:
     TEST_DIRS += ["tests"]
 
-DIRS += ["/third_party/fmt"]
+DIRS += [
+    "/third_party/fmt",
+    "/third_party/simdjson",
+]
diff --git a/netwerk/protocol/http/moz.build b/netwerk/protocol/http/moz.build
index d1cd8ce368dd..b46b00b8ded4 100644
--- a/netwerk/protocol/http/moz.build
+++ b/netwerk/protocol/http/moz.build
@@ -246,4 +246,7 @@ XPCOM_MANIFESTS += [
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/netwerk/protocol/websocket/moz.build b/netwerk/protocol/websocket/moz.build
index da52adc74881..571ce0c01f43 100644
--- a/netwerk/protocol/websocket/moz.build
+++ b/netwerk/protocol/websocket/moz.build
@@ -65,6 +65,9 @@ LOCAL_INCLUDES += [
 ]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
 
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
