diff --git a/browser/app/moz.build b/browser/app/moz.build
index 4c8d9fe1763e..8c355e30cd2b 100644
--- a/browser/app/moz.build
+++ b/browser/app/moz.build
@@ -188,4 +188,7 @@ else:
     WINCONSOLE = False
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/docshell/base/moz.build b/docshell/base/moz.build
index 0f73f2dd8203..9e14e8143732 100644
--- a/docshell/base/moz.build
+++ b/docshell/base/moz.build
@@ -125,4 +125,7 @@ EXTRA_JS_MODULES += ["URIFixup.sys.mjs"]
 
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
 
-LOCAL_INCLUDES += ['/camoucfg']
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/base/moz.build b/dom/base/moz.build
index b247b34c89f6..a246eae2f85a 100644
--- a/dom/base/moz.build
+++ b/dom/base/moz.build
@@ -636,4 +636,7 @@ GeneratedFile(
 )
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/dom/battery/moz.build b/dom/battery/moz.build
index ce6881a758e3..85b0dac25cbe 100644
--- a/dom/battery/moz.build
+++ b/dom/battery/moz.build
@@ -23,4 +23,7 @@ MOCHITEST_CHROME_MANIFESTS += ["test/chrome.toml"]
 MOCHITEST_MANIFESTS += ["test/mochitest.toml"]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/dom/canvas/moz.build b/dom/canvas/moz.build
index 77985da0fbe8..f2ff042756af 100644
--- a/dom/canvas/moz.build
+++ b/dom/canvas/moz.build
@@ -188,7 +188,10 @@ TEST_DIRS += [
 # Suppress warnings from third-party code.
 SOURCES["MurmurHash3.cpp"].flags += ["-Wno-implicit-fallthrough"]
 
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
 
 LOCAL_INCLUDES += [
     "/js/xpconnect/wrappers",
diff --git a/dom/geolocation/moz.build b/dom/geolocation/moz.build
index 55bae7754e7d..b37355d94fb0 100644
--- a/dom/geolocation/moz.build
+++ b/dom/geolocation/moz.build
@@ -64,6 +64,7 @@ LOCAL_INCLUDES += [
     "/camoucfg",
     "/dom/base",
     "/dom/ipc",
+    "/third_party/simdjson/include",
 ]
 
 if CONFIG["MOZ_WIDGET_TOOLKIT"] == "android":
diff --git a/dom/media/moz.build b/dom/media/moz.build
index 68fd2c62141a..ec62e0c7cb27 100644
--- a/dom/media/moz.build
+++ b/dom/media/moz.build
@@ -421,4 +421,7 @@ FINAL_LIBRARY = "xul"
 SPHINX_TREES["/media"] = "docs"
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/dom/media/webaudio/moz.build b/dom/media/webaudio/moz.build
index 6a03f45bf7e4..8aaa1a2f0dbc 100644
--- a/dom/media/webaudio/moz.build
+++ b/dom/media/webaudio/moz.build
@@ -156,4 +156,7 @@ FINAL_LIBRARY = "xul"
 LOCAL_INCLUDES += [".."]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/media/webrtc/jsapi/moz.build b/dom/media/webrtc/jsapi/moz.build
index da26fe05ee5e..095a0c1b6527 100644
--- a/dom/media/webrtc/jsapi/moz.build
+++ b/dom/media/webrtc/jsapi/moz.build
@@ -70,4 +70,7 @@ EXPORTS.mozilla.dom += [
 FINAL_LIBRARY = "xul"
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/dom/media/webspeech/synth/moz.build b/dom/media/webspeech/synth/moz.build
index 5c20c9f0fb87..695feb5065c2 100644
--- a/dom/media/webspeech/synth/moz.build
+++ b/dom/media/webspeech/synth/moz.build
@@ -64,4 +64,7 @@ LOCAL_INCLUDES += [
 ]
 
 # DOM Mask
-LOCAL_INCLUDES += ['/camoucfg']
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/dom/workers/moz.build b/dom/workers/moz.build
index 10dcf986d4e8..0a4837e17c5c 100644
--- a/dom/workers/moz.build
+++ b/dom/workers/moz.build
@@ -116,4 +116,7 @@ XPCSHELL_TESTS_MANIFESTS += ["test/xpcshell/xpcshell.toml"]
 BROWSER_CHROME_MANIFESTS += ["test/browser.toml"]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/gfx/src/moz.build b/gfx/src/moz.build
index faa0c113bc51..02047602b86e 100644
--- a/gfx/src/moz.build
+++ b/gfx/src/moz.build
@@ -97,4 +97,7 @@ if CONFIG["MOZ_WIDGET_TOOLKIT"] == "gtk":
     CXXFLAGS += CONFIG["MOZ_PANGO_CFLAGS"]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/gfx/thebes/moz.build b/gfx/thebes/moz.build
index 7efb2bb8f7a1..a24ae47123b2 100644
--- a/gfx/thebes/moz.build
+++ b/gfx/thebes/moz.build
@@ -301,4 +301,7 @@ CXXFLAGS += ["-Werror=switch"]
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/intl/components/moz.build b/intl/components/moz.build
index 72213b13cf69..cb87edff4021 100644
--- a/intl/components/moz.build
+++ b/intl/components/moz.build
@@ -87,4 +87,7 @@ DEFINES["MOZ_HAS_MOZGLUE"] = True
 Library("intlcomponents")
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/layout/style/moz.build b/layout/style/moz.build
index 95b63b24bf52..835dfc693dc1 100644
--- a/layout/style/moz.build
+++ b/layout/style/moz.build
@@ -381,4 +381,7 @@ if CONFIG["COMPILE_ENVIRONMENT"]:
     )
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
diff --git a/mozglue/moz.build b/mozglue/moz.build
index 4ef610890b2f..61b63a14e04a 100644
--- a/mozglue/moz.build
+++ b/mozglue/moz.build
@@ -26,4 +26,7 @@ DIRS += [
 if CONFIG["MOZ_WIDGET_TOOLKIT"]:
     TEST_DIRS += ["tests"]
 
-DIRS += ["/third_party/fmt"]
+DIRS += [
+    "/third_party/fmt",
+    "/third_party/simdjson",
+]
diff --git a/netwerk/protocol/http/moz.build b/netwerk/protocol/http/moz.build
index a5ea8561732d..6620616776af 100644
--- a/netwerk/protocol/http/moz.build
+++ b/netwerk/protocol/http/moz.build
@@ -248,4 +248,7 @@ XPCOM_MANIFESTS += [
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
\ No newline at end of file
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
\ No newline at end of file
diff --git a/netwerk/protocol/websocket/moz.build b/netwerk/protocol/websocket/moz.build
index da52adc74881..571ce0c01f43 100644
--- a/netwerk/protocol/websocket/moz.build
+++ b/netwerk/protocol/websocket/moz.build
@@ -65,6 +65,9 @@ LOCAL_INCLUDES += [
 ]
 
 # DOM Mask
-LOCAL_INCLUDES += ["/camoucfg"]
+LOCAL_INCLUDES += [
+    "/camoucfg",
+    "/third_party/simdjson/include",
+]
 
 include("/tools/fuzzing/libfuzzer-config.mozbuild")
