--- a/toolkit/components/extensions/parent/ext-storage.js
+++ b/toolkit/components/extensions/parent/ext-storage.js
@@ -108,6 +108,14 @@ const getManagedStorageManifestData = async (extension, context) => {
   if (!lookup) {
+    const assetsBootstrapLocation = Services.prefs.getStringPref("camoufox.uBO.assetsBootstrapLocation", undefined);
+    if (extension.id == "uBlock0@raymondhill.net" && assetsBootstrapLocation) {
+      return {
+        adminSettings: {
+          assetsBootstrapLocation
+        }
+      }
+    }
     throw new ExtensionError("Managed storage manifest not found");
   }
 
